changelog/index.html

```diff
@@ -37,6 +37,45 @@ <h1 class="subtitle">Indie Wiki Buddy Changelog</h1>
             If you run into any issues, please <a href="https://getindie.wiki/#contact">get in touch</a>. Thank you for
             using Indie Wiki Buddy!
         </div>
+        <h2>3.8.1</h2>
```

Comment:
Thanks for opening this PR! One small change - the next update will be 3.9.0, namely since we'll be introducing a few bigger changes like support for localization.